cookiecutter.json

```diff
@@ -21,5 +21,6 @@
         "Apache Software License 2.0",
         "GNU LGPL v3.0",
         "GNU GPL v3.0"
-    ]
+    ],
+    "twitter_handle": "none"
```

Comment:
I do wonder about this ... it does feel a little funny to call out `twitter` here. I know it's something a lot of us use .... I'm not sure ...

Reply:
Yeah agreed, it's so out of place. Since this is a field that is silently suppressed on the hub, I think we should just exclude it here and alert users to its existence elsewhere - probably on the validation side.